azplugins/analyze.py

```diff
@@ -18,6 +21,58 @@
 
 from . import _azplugins
 
+class group_velocity(hoomd.compute._compute):
+    r"""Group center-of-mass velocity compute
+
+    Args:
+        group (:py:mod:`hoomd.group`): Group to compute velocity of.
+        suffix (str): Suffix to attach to logged quantities.
+
+    This computes the center-of-mass velocity of a group:
+
+    .. math::
+
+        \mathbf{v}_{\rm cm} = \dfrac{\sum_i m_i \mathbf{v}_i}{\sum_i m_i}
+
+    where :math:`\mathbf{v}_i` is the velocity and and :math:`m_i` is the mass
+    of particle *i* in the group.
+
+    The components of the result are exposed as loggable quantities ``vx``,
+    ``vy``, and ``vz`` with ``suffix`` appended. By default, ``suffix`` is
+    ``_name`` where ``name`` is the name of the ``group``, like ``_all`` for
+    :py:class:`hoomd.group.all`. However, a custom suffix may be specified; the
+    only requirement is that the same suffix cannot be used more than once. You
+    can save these results using :py:class:`hoomd.analyze.log`.
+
+    Example::
+
+        azplugins.analyze.group_velocity(hoomd.group.all())
+        hoomd.analze.log(filename='velocity.dat', quantities=['vx_all'], period=10)
```

Comment:
Could you add an example with a custom name? Just to be overly explicit for users that are less experienced with the logger? 

Also, there is a typo "analze" -> "analyze".

Reply:
Resolved on #64